example/color_cycles.py

```diff
@@ -18,14 +19,13 @@
 cycle = plt.rcParams["axes.prop_cycle"].by_key()["color"]
 
 # plot the color cycles and corresponding color codes.
-for i, c in enumerate(cycle):
-    ax.plot(x, y + offset * i, label="\\{}".format(c), color=c, linestyle="-")
-
+for i, hex in enumerate(cycle):
+    ax.plot(x, y + offset * i, label=Colors(hex).name, color=hex, linestyle="-")
```

Comment:
@sbillinge  You are correct. 

I found out that we do not need `get_color_name`. Python's enum provide `Colors(hex).name` where we can get the name from a string. Didn't know about this feature but certainly saved many lines of code.